ddl/index.go

```diff
@@ -679,49 +691,64 @@ func onDropIndex(t *meta.Meta, job *model.Job) (ver int64, _ error) {
 		// Finish this job.
 		if job.IsRollingback() {
 			job.FinishTableJob(model.JobStateRollbackDone, model.StateNone, ver, tblInfo)
-			job.Args[0] = indexInfo.ID
+			job.Args[0] = indexesIDs
 			// the partition ids were append by convertAddIdxJob2RollbackJob, it is weird, but for the compatibility,
 			// we should keep appending the partitions in the convertAddIdxJob2RollbackJob.
 		} else {
 			job.FinishTableJob(model.JobStateDone, model.StateNone, ver, tblInfo)
-			job.Args = append(job.Args, indexInfo.ID, getPartitionIDs(tblInfo))
+			job.Args = append(job.Args, indexesIDs, getPartitionIDs(tblInfo))
 		}
 	default:
-		err = ErrInvalidDDLState.GenWithStackByArgs("index", indexInfo.State)
+		err = ErrInvalidDDLState.GenWithStackByArgs("index", idxInfos[0].State)
 	}
 	return ver, errors.Trace(err)
 }
 
-func checkDropIndex(t *meta.Meta, job *model.Job) (*model.TableInfo, *model.IndexInfo, error) {
+func checkDropIndexes(t *meta.Meta, job *model.Job) (*model.TableInfo, []*model.IndexInfo, error) {
 	schemaID := job.SchemaID
 	tblInfo, err := getTableInfoAndCancelFaultJob(t, job, schemaID)
 	if err != nil {
 		return nil, nil, errors.Trace(err)
 	}
 
-	var indexName model.CIStr
-	if err = job.DecodeArgs(&indexName); err != nil {
-		job.State = model.JobStateCancelled
-		return nil, nil, errors.Trace(err)
-	}
-
-	indexInfo := tblInfo.FindIndexByName(indexName.L)
-	if indexInfo == nil {
-		job.State = model.JobStateCancelled
-		return nil, nil, ErrCantDropFieldOrKey.GenWithStack("index %s doesn't exist", indexName)
+	var indexNames []model.CIStr
+	var ifExists []bool
+	if err = job.DecodeArgs(&indexNames, &ifExists); err != nil {
+		// Compatible with the previous function: DropIndex
```

Comment:
I think we‘d better add a new operation `DropIndexes` to handle it. Otherwise, we need to handle the compatible issue on[ rollback'Args](https://github.com/pingcap/tidb/pull/20457/files#diff-8b1b64b0939bd0f94f247a034802a0f4eed8e6537f77a2b3bde1747ffef0d63bR286).

Reply:
@zimulala Hello, I added an action in the parser: https://github.com/pingcap/parser/pull/1185.
How can I test my code when TiDB has not updated the Parser version ？

Reply:
We can test it using `go mod edit -replace ` to replace the "pingcap/parser" package in "go.mod". You can refer to [using-go-modules](https://blog.golang.org/using-go-modules).

Reply:
I will test my code locally now 👌